app/templates/frameworks/reuse_declaration.html

```diff
@@ -28,38 +28,45 @@
 {% endblock %}
 
 {% block main_content %}
-  <div class="grid-row framework-dashboard">
+  <div class="grid-row">
     <div class="column-two-thirds">
       {% with
          heading = self.page_title_inner(),
          smaller = True
       %}
         {% include "toolkit/page-heading.html" %}
       {% endwith %}
+
```

Comment:
The general pattern we use for validation is that if there are multiple questions on the page then you get this banner with a list of the things with problems.

But for just one question on a page we don't have the banner, just the red bar and message next to the question itself.

(Grab Ralph or Tom if they have a second to confirm, but I'm pretty sure this is right.)